build.gradle.kts

```diff
@@ -61,4 +65,8 @@ allprojects {
             exceptionFormat = TestExceptionFormat.FULL
         }
     }
+
+    afterEvaluate {
+        rootProject.dotenv?.let { dotenv -> injectInto(JavaExec::class, Test::class) environmentsFrom dotenv }
+    }
 }
```

Comment:
Since I see you added the job to generate the scaladoc I suggest to add this to make the version of the project be aligned with git history (applying the `org.danilopianini.git-sensitive-semantic-versioning:3.1.7` plugin). In this way when scala doc is generated should reflects the version of the project just released.

```
/* Set the project version based on the git history. */
gitSemVer {
    assignGitSemanticVersion()
}
```